### PR TITLE
Added Doug Finke's ImportExcel module

### PR DESCRIPTION
**Pull request by dfinke**
Let me know of changes needed. Thanks for putting this together

### CODE REVIEW

File: Modules/ImportExcel.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Name: ImportExcel
+
+## Description:
+PowerShell module to import/export Excel spreadsheets, without Excel Edit
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| PowerShell 4.0 | [Doug Finke](https://twitter.com/dfinke) | [Github](https://github.com/dfinke/ImportExcel) | May 2015 | powershell, excel |
```
Comment (ClaudioESSilva) [inline Modules/ImportExcel.md:8]: Please add your blog URL on the same cell of Twitter.

File: Modules/ImportExcel.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* On the PowerShell Gallery - [ImportExcel](https://www.powershellgallery.com/packages/ImportExcel)
```
Comment (ClaudioESSilva) [inline Modules/ImportExcel.md]: Please, add a new cell on the table with title PowerShell gallery and the URL on the value.
I will go through others modules add this too so it becomes coherent.
It's a good addition. Thanks.

File: Modules/ImportExcel.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+____
+
+### Shared blog posts:
+* [Introducing the PowerShell Excel Module](https://blogs.technet.microsoft.com/heyscriptingguy/2015/11/25/introducing-the-powershell-excel-module-2/)
```
Comment (ClaudioESSilva) [inline Modules/ImportExcel.md]: Add your name and the (b | t) before the list of posts. You can check this on any other module md file.

File: Modules/ImportExcel.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* [Import-Html Use PowerShell to Export Html Tables to Excel](https://dfinke.github.io/2016/import-html-use-powershell-to-export-html-tables-to-excel/)
+* [Read and write Excel spreadsheets with the ImportExcel PowerShell module](https://4sysops.com/archives/read-and-write-excel-spreadsheets-with-the-importexcel-powershell-module/)
+* [The ImportExcel PowerShell Module](https://www.altitude365.com/2016/01/08/importexcel-powershell/)
+____
+
+### Shared videos:
+
```
Comment (ClaudioESSilva) [inline Modules/ImportExcel.md]: Same as before. Add your name and (b | t)

Also order the subtitles alphabetically. Switch tracking ups with the last one.

Curiosity: why the last has the [image] and the others is empty?

File: readme.md [hunk 5]
```diff
@@ -31,6 +31,7 @@ Please, read the [CONTRIBUTING.md](https://github.com/ClaudioESSilva/PowerShellM
   * [dbatools](https://github.com/ClaudioESSilva/PowerShellModulesCentral/blob/master/Modules/dbatools.md)
   * [DRSRule](https://github.com/ClaudioESSilva/PowerShellModulesCentral/blob/master/Modules/DRSRule.md)
   * [FixDaDatabase](https://github.com/ClaudioESSilva/PowerShellModulesCentral/blob/master/Modules/FixDaDatabase.md)
+  * [ImportExcel](https://github.com/ClaudioESSilva/PowerShellModulesCentral/blob/master/Modules/ImportExcel.md)
```
Comment (ClaudioESSilva) [inline readme.md]: You can discard the changes om readme.md file from the PR. I will update it when merged

**Review by ClaudioESSilva — CHANGES_REQUESTED** [review]
Thanks Doug. Few request changes.

**Comment by ClaudioESSilva** [comment]
Thanks for your contribution @dfinke ! Asked for some minor changes. Let me know if you have any question

**Comment by ClaudioESSilva** [comment]
@dfinke If you prefer I can do those minor adjustments. Let me know. Thanks

**Comment by dfinke** [comment]
Please do. Thank you.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ClaudioESSilva** [comment]
Thanks for your contribution @dfinke !

**Pull request merged**